packages/cadl-ranch-specs/http/type/property/value-types/main.tsp

```diff
@@ -183,3 +183,31 @@ model NeverProperty is ModelTemplate<never>;
 @route("/never")
 @operationGroup
 interface Never extends ModelOperations<NeverProperty, "<don't include this property>"> {}
+
+// Test a model with unknown and string
+@doc("Model with a property unknown, and the data is a string.")
```

Comment:
i feel this test would be better as a top level file under `type`, i.e. `Type.Unknown` since we want to test how `unknown` interacts with other types. I think it'd be good to still have one basic test for `unknown` in property too, just to check how it interacts with a property type

Reply:
We do need to add a lot more test specially for all the scalars too